charts/scality-mountpoint-s3-csi-driver/values.yaml

```diff
@@ -3,10 +3,10 @@
 # Declare variables to be passed into your templates.
 
 image:
-  # Our regional ECR repositories are labelled `eks/aws-s3-csi-driver`.
+  # Our regional ECR repositories are labelled `eks/scality-s3-csi-driver`.
   # See https://docs.aws.amazon.com/eks/latest/userguide/add-ons-images.html for the per-region registry list.
-  # Example: `602401143452.dkr.ecr.us-east-1.amazonaws.com/eks/aws-s3-csi-driver` for us-east-1.
-  repository: public.ecr.aws/mountpoint-s3-csi-driver/aws-mountpoint-s3-csi-driver
+  # Example: `602401143452.dkr.ecr.us-east-1.amazonaws.com/eks/scality-s3-csi-driver` for us-east-1.
+  repository: public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
   tag: "v1.13.0"
```

Comment:
same here we won't use this registry and this version. And the registry alias `mountpoint-s3-csi-driver` is not owned by us

Reply:
Changed in the next PR: https://github.com/scality/mountpoint-s3-csi-driver/pull/29